### PR TITLE
feat: support iOS 13 auth-policy constants

### PR DESCRIPTION
**Pull request by hansemannn**
**JIRA**: https://jira.appcelerator.org/browse/MOD-2532

### CODE REVIEW

**Comment by sgtcoolguy** [comment]
It'd be good to add unit tests for the new constants...

**Comment by hansemannn** [comment]
@sgtcoolguy Added

File: apidoc/Identity.yml [hunk 1]
```diff
@@ -212,6 +212,23 @@ properties:
...
     platforms: [iphone, ipad, android]
     since: "6.1.0"
+
+  - name: AUTHENTICATION_POLICY_BIOMETRICS_OR_WATCH
+    summary: Device owner was authenticated using a biometric method or the Apple Watch.
+    type: Number
+    permission: read-only
+    platforms: [iphone, ipad]
+    osver: {ios: {min: "13.0"}}
+    since: "8.3.0"
```
Comment (vijaysingh-axway) [inline apidoc/Identity.yml]: @hansemannn As we are planning to include this is 8.2.1, can you change since to "8.2.1".

File: apidoc/Identity.yml [hunk 2]
```diff
@@ -212,6 +212,23 @@ properties:
...
+
+  - name: AUTHENTICATION_POLICY_WATCH
+    summary: Device owner was authenticated using the Apple Watch.
+    type: Number
+    permission: read-only
+    platforms: [iphone, ipad]
+    osver: {ios: {min: "13.0"}}
+    since: "8.3.0"
```
Comment (vijaysingh-axway) [inline apidoc/Identity.yml]: Same as above.

File: ios/Classes/TiIdentityModule.m [hunk 3]
```diff
@@ -274,5 +274,9 @@ - (NSDictionary *)deviceCanAuthenticate:(id)unused
 
 MAKE_SYSTEM_PROP(AUTHENTICATION_POLICY_BIOMETRICS, LAPolicyDeviceOwnerAuthenticationWithBiometrics);
 MAKE_SYSTEM_PROP(AUTHENTICATION_POLICY_PASSCODE, LAPolicyDeviceOwnerAuthentication);
+#ifdef __IPHONE_13_0
```
Comment (vijaysingh-axway) [inline ios/Classes/TiIdentityModule.m]: Shouldn't it be like #if __IPHONE_OS_VERSION_MAX_ALLOWED >= 130000

**Pull request merged**